### DISCUSSION TITLE
effort

### DISCUSSION THREAD

**Discussion by kubraaksux** [bounswe/bounswe2023group1#64] [Q&A]
is effort label efficient for you?
some groups have more effort levels. for ex: 1 to 7.
maybe we can use effort levels more than 3? or is it enough for now?

**Comment by HarunErgen** [comment]
In my opinion the number of levels used is less important than ensuring that everyone involved has a shared understanding of what each level means. Does the person assigned to the task determine the level of effort according to their own opinion? Because effort is very subjective.

**Reply by kubraaksux** [reply]
I agree with you. But to make it more concrete, adding something to sidebar is low effort, or maybe revision can be low effort too if everything goes well. But designing of mock-ups can be higher effort or creating whole requirements. Effort level meaning is not lined strictly due to subjectivity but it can be measured with predictions about the overall issues and tasks
